ExcelMapper/ExcelMapper.cs

```diff
@@ -324,15 +324,20 @@ IEnumerable Fetch(ISheet sheet, Type type)
 
                         if (cell != null && (!SkipBlankRows || !IsCellBlank(cell)))
                         {
-                            var cellValue = GetCellValue(cell, col.Value);
-
-                            try
-                            {
-                                col.Value.SetProperty(o, cellValue);
-                            }
-                            catch (Exception e)
+                            foreach (var ci in col.Value)
                             {
-                                throw new ExcelMapperConvertException(cellValue, col.Value.PropertyType, i, col.Key, e);
+                                if (!ci.Direction.HasFlag(ColumnInfoDirection.Cell2Prop))
```

Comment:
This could be moved to a Where clause on the foreach loop. Likewise in a few other places where if is used inside a loop.